stylelint.config.js

```diff
@@ -1,34 +0,0 @@
-module.exports = {
-  plugins: ['stylelint-scss'],
-  extends: ['stylelint-config-standard', 'stylelint-config-sass-guidelines'],
-  ignoreFiles: ['src/brand-styles/**'],
-  rules: {
-    'selector-class-pattern': [
-      '^[a-z0-9\\-_]+$',
-      {
-        message:
-          'Selector should be written in lowercase with hyphens (selector-class-pattern)'
-      }
-    ],
-    'number-leading-zero': 'never',
-    'max-nesting-depth': 3,
-    'selector-pseudo-class-no-unknown': [
-      true,
-      {
-        ignorePseudoClasses: ['local']
-      }
-    ],
-    'at-rule-no-unknown': [
-      true,
-      {
-        ignoreAtRules: ['include', 'mixin']
-      }
-    ],
-    'property-no-unknown': [
-      true,
-      {
-        ignoreProperties: ['composes']
-      }
-    ]
-  }
-};
```

Comment:
Why is this outdated? Aren't we still using this?

Reply:
No, this is no longer used. The existing setup is for .scss files but since we switched to css-in-js, it doesn't apply anymore. There are ways to get stylelint to work with css-in-js, so maybe setting that up would be better?

Reply:
Hm yeah, I've been looking into that. The styled-components plugin for stylelint works, according to [this issue](https://github.com/emotion-js/emotion/issues/185). But I'm fine with deleting all this for now and setting it up fresh once we have time.